source/reference/limitations.txt

```diff
@@ -0,0 +1,73 @@
+.. _c2c-limitations:
+
+===========
+Limitations
+===========
+
+.. include:: /includes/preview-warning
+
+{+c2c-full-product-name+} is under active development. This preview
+release has several limitations.
+
+General Limitations
+-------------------
+
+- The minimum supported version is MongoDB 6.0.
+- The minimum support Feature Compatibility Version is 6.0. 
+- MongoDB Community Edition is not supported.
+- The source and destination clusters must have the same release
+  version.
+- Environmental assumptions are not validated as pre-checks. 
+- The Primary cluster cannot be changed while syncing.
+- If write blocking is disabled,the client must prevent writes to the
+  source cluster after starting the commit process.
+- DDL operations are not supported during syncing. 
+- Filtered synchronization is not supported.
+- Network compression is not supported.
+- applyOps operations from the source cluster are not supported.
```

Comment:
Can we say 'applyOps' command and link to https://www.mongodb.com/docs/manual/reference/command/applyOps/? Some people think of "applyOps" as the name of the process where secondary nodes apply operations from the primary during normal steady state replication. I want to make sure we clarify this for users